File: PRAKTIKUM MODUL 6/PRAK604-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>
#include <string.h>

int main() {
    char asli[100];
    char palsu[100];

    scanf("%[^\n]%*c", &asli);   
    scanf("%[^\n]%*c", &palsu);  

    int code, mas, x, i = 0, j = 0;
    code = strlen(asli);
    mas = strlen(palsu);

    if (code != mas) {
        printf("Panjang kalimat berbeda, pesan palsu");
    } else {
        for (x = 0; x < code; x++) {
            if (asli[x] == ' ' && palsu[x] == ' ') {  
                printf(" ");  
            } else if (asli[x] == ' ' && palsu[x] == ' ') {
                continue;
            } else if (asli[x] == palsu[x]) {
                printf("*");
                i++;
            } else {
                printf("#");
                j++;
            }
        }

        printf("\n* = %d", i);  
        printf("\n# = %d", j);  

        if (i >= j) {
            printf("\nPesan Asli");
        } else if (i < j) {
            printf("\nPesan Palsu");
        }
    }
    return 0;
}

File: PRAKTIKUM MODUL 6/PRAK604-2410817310009-Muhammad_Ghazi_Rakhmadi.py
asli = input()
palsu = input()

g = 0  
z = 0 

if len(asli) != len(palsu):
    print('Panjang kalimat berbeda, pesan palsu')
else:
    for i in range(len(asli)):
        if asli[i] == ' ' and palsu[i] == ' ':
            print(' ', end='') 
        elif asli[i] == ' ' and palsu[i] == ' ':
            continue  
        elif asli[i] == palsu[i]:
            print('*', end='')
            g += 1
        else:
            print('#', end='')
            z += 1

    print('\n* = {}'.format(g))
    print('# = {}'.format(z))

    if g >= z:
        print('Pesan Asli')
    else:
        print('Pesan Palsu')